tests/config.yml

```diff
@@ -7,3 +7,5 @@ vcd:
   catalog: 'cse'
   template: 'k8s-u.ova'
   verify: False
+  role_name: System Administrator
```

Comment:
does it need to be enclosed in quotes due to the space?

Reply:
No. I didn't have to enclose it in quotes. The unit test picked the whole role_name even when it had spaces in it.

Reply:
ok